# This service allows the user to set the current bounds in the point cloud
#  frame so that we can narrow our search region for objects

# We will form our bounds as follows: create a rectangular prism that is
#  centered at the origin, and will extend along the x,
#  y, and z directions. Then, we will transform this box to somewhere else
#  in the point cloud frame using a homogeneous transform


float64[3] widths   # width of prism along x, y, and z directions

float64[3] trans    # translation component of homogeneous transform
float64[4] quat     # rotational component of homogeneous transform

---

int64 ret           # Whether we were successfully able to change bounds
string msg          # Any error message associated with it

